extensions/amp-a4a/0.1/callout-vendors.js

```diff
@@ -60,13 +60,13 @@ const RTC_VENDORS = jsonConfiguration({
   },
   prebidappnexus: {
     url:
-      'https://prebid.adnxs.com/pbs/v1/openrtb2/amp?tag_id=PLACEMENT_ID&w=ATTR(width)&h=ATTR(height)&ow=ATTR(data-override-width)&oh=ATTR(data-override-height)&ms=ATTR(data-multi-size)&slot=ATTR(data-slot)&targeting=TGT&curl=CANONICAL_URL&timeout=TIMEOUT&adcid=ADCID&purl=HREF',
+      'https://prebid.adnxs.com/pbs/v1/openrtb2/amp?tag_id=PLACEMENT_ID&w=ATTR(width)&h=ATTR(height)&ow=ATTR(data-override-width)&oh=ATTR(data-override-height)&ms=ATTR(data-multi-size)&slot=ATTR(data-slot)&targeting=TGT&curl=CANONICAL_URL&timeout=TIMEOUT&adcid=ADCID&purl=HREF&gdpr_consent=ATTR(consentString)',
```

Comment:
The macro is `CONSENT_STRING`. 
Please also include the `CONSENT_STRING` to the `macros` declaration below. Thanks. 
You can refer the usage here for an example. 
https://github.com/ampproject/amphtml/blob/5309266d912717789127b931ff73c6d00c835cee/extensions/amp-a4a/0.1/callout-vendors.js#L116-L128

Reply:
Thanks - fixed.